Packs/Core/Playbooks/Playbook-Large_Upload.yml

```diff
@@ -0,0 +1,3868 @@
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1840,
+          "y": 470
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 448fefe4-c69a-45fd-899c-6e138ce31e92
+    type: condition
+    task:
+      id: 448fefe4-c69a-45fd-899c-6e138ce31e92
+      version: -1
+      name: Alert Investigation Should Continue?
+      description: |-
+        The playbook detected similar alerts that were closed as false positives previously.
+
+        Review the alert details and similar closed false positive alerts to determine whether this Cortex XDR alert should be further investigated.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "7"
+      "Yes":
+      - "6"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1580,
+          "y": 640
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: e2253abd-a716-47b4-8475-796c25442726
+    type: title
+    task:
+      id: e2253abd-a716-47b4-8475-796c25442726
+      version: -1
+      name: Enrichment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "8"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 810
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: f53c8abc-04b8-4315-8942-eaf621e1b202
+    type: title
+    task:
+      id: f53c8abc-04b8-4315-8942-eaf621e1b202
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1840,
+          "y": 5060
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 932a0964-da64-41c3-84c4-e7a392b584f4
+    type: condition
+    task:
+      id: 932a0964-da64-41c3-84c4-e7a392b584f4
+      version: -1
+      name: Check Uploaded Data Volume
+      description: Determines if the amount of data uploaded to an external host exceeds the defined threshold amount.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "7"
```

Comment:
Same as the above - maybe we should add close alert here?

Reply:
done, added task number 59 - Close Alert - Resolve As Other - closes the alert as 'other' if the uploaded data volume did not exceed the threshold set in the playbook input.